internal/provider/resource_integration_test.go

```diff
@@ -250,3 +251,150 @@ resource "juju_integration" "a" {
 }
 `, srcModelName, dstModelName, viaCIDRs)
 }
+
+func TestAcc_ResourceIntegrationWithMultipleConsumers_Stable(t *testing.T) {
+        if testingCloud != LXDCloudTesting {
```

Comment:
s/Stable/Edge/ to name test as a test of the current code, not the stable version, currently 0.8.0

Reply:
Modified as per the suggestion